Thu Nov  3 18:27:37 2022 - INFO  - Start environmental preparation.
Thu Nov  3 18:29:00 2022 - INFO  - End of environmental preparation!
Thu Nov  3 18:29:03 2022 - INFO  - Start to run test.
Failed to restart ceph-mds.target: Unit ceph-mds.target not found.
Thu Nov  3 18:29:06 2022 - ERROR - ceph-mds.target restart failed
Thu Nov  3 18:29:08 2022 - ERROR - ../common/common_lib.sh line 34
Unit ceph-mds.target could not be found.
Thu Nov  3 18:29:19 2022 - ERROR - ceph-mds.target restart failed
Thu Nov  3 18:29:22 2022 - ERROR - ../common/common_lib.sh line 37
Failed to stop ceph-mds.target: Unit ceph-mds.target not loaded.
Thu Nov  3 18:29:25 2022 - ERROR - ceph-mds.target stop failed
Thu Nov  3 18:29:27 2022 - ERROR - ../common/common_lib.sh line 39
Unit ceph-mds.target could not be found.
Thu Nov  3 18:29:39 2022 - ERROR - ceph-mds.target stop failed
Thu Nov  3 18:29:42 2022 - ERROR - ../common/common_lib.sh line 42
Failed to start ceph-mds.target: Unit ceph-mds.target not found.
Thu Nov  3 18:29:45 2022 - ERROR - ceph-mds.target start failed
Thu Nov  3 18:29:47 2022 - ERROR - ../common/common_lib.sh line 44
Unit ceph-mds.target could not be found.
Thu Nov  3 18:29:58 2022 - ERROR - ceph-mds.target start failed
Thu Nov  3 18:30:00 2022 - ERROR - ../common/common_lib.sh line 47
Failed to get unit file state for ceph-mds.target: No such file or directory
Thu Nov  3 18:30:03 2022 - INFO  - Unit is indirect, ignoring.
Failed to start ceph-mds.target: Unit ceph-mds.target not found.
Thu Nov  3 18:30:07 2022 - ERROR - oe_test_target_ceph-mds.sh line 33
Unit ceph-mds.target could not be found.
Thu Nov  3 18:30:09 2022 - ERROR - oe_test_target_ceph-mds.sh line 35
Thu Nov  3 18:30:12 2022 - INFO  - End of the test.
Thu Nov  3 18:30:14 2022 - ERROR - failed to execute the case.
Thu Nov  3 18:30:17 2022 - INFO  - start environment cleanup.
Traceback (most recent call last):
  File "/root/mugen/mugen-riscv/libs/locallibs/rpm_manage.py", line 173, in <module>
    exitcode = rpm_remove(args.node, args.pkgs, args.tempfile)
  File "/root/mugen/mugen-riscv/libs/locallibs/rpm_manage.py", line 144, in rpm_remove
    with open(tmpfile, "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'Last metadata expiration check: 0:10:38 ago on 2022年11月03日 星期四 18时18分18秒.\nNo match for argument: ceph-mds\nError: Unable to find a match: ceph-mds'
Thu Nov  3 18:30:28 2022 - INFO  - Finish environment cleanup!
